README.adoc

```diff
@@ -91,6 +92,66 @@ When a rule matches, the Authorizer will return that rule’s evaluation as a de
 When no rules match, then by default it is denied.
 So, if ACL is not defined for a resource type then it automatically gets denied.
 
+== ManagedKafkaCreateTopicPolicy
+
+The objective of this policy is to:
+
+* validate the topic replication factor (default allows 3 only)
+* validate number of partitions in the cluster
+* validate the create topic configs
+
+Here are the configuration description:
+
+[cols="2,1,5"]
+|===
+|property |default value |description
+
+|kas.policy.create-topic.max.partitions |-1 |Specifies the upper limit of partitions that should be allowed in the cluster. If this property is not specified, the default of `-1` implies no limit.
+|kas.policy.create-topic.partition-counter.timeout-seconds |10 |Specifies the duration in seconds to use as a timeout when listing and describing topics. Defaults to 10.
+|kas.policy.create-topic.partition-counter.private-topic-prefix |"" |Specifies the prefix used to match against topic name to detect private/internal topics. Defaults to blank - no prefix.
+|kas.policy.create-topic.partition-counter.schedule-interval-seconds |15 |Specifies the interval (in seconds) between partition counts. Defaults to 15.
+|kas.policy.create-topic.partition-limit-enforced |false |`true` disables the enforcement of the partition limit. Defaults to `false` (limit enforced).
```

Comment:
Is this right, the description is the opposite mapping to what I expect from the property name. 

I expect `true` -> `Enforced`, `false` -> `Disabled` but your description is the opposite.

Reply:
You're right! Updated.